README.md

```diff
@@ -94,48 +50,33 @@ Navigate to `ServicePulse\src\ServicePulse.Host\vue` and:
 
 If `npm run dev` fails with an error related to git submodules not being correctly configured, run the `npm install` command again to ensure all required dependencies are available, and then run `npm run dev`.
 
-#### Step 4 - open the browser
+#### Step 2 - open the browser
 
-After doing the above steps, navigate to `http://localhost:1331` to see the ServicePulse application.
+After doing the above steps, navigate to the URL presented by the execution of the Vue.js application to see the ServicePulse application.
 
 ### Provided npm scripts
 
 #### Vue.js
 
-- `dev` - runs `vite` that starts development server doing hot reload over source files
+- `dev` - runs `vite` that starts the development server doing hot reload over source files
 - `build` - runs build script that outputs files to `..\app` folder
-- `lint` - checks with eslint all vue, ts and js files
+- `lint` - checks with eslint all vue, ts, and js files
 - `type-check` - runs TypeScript in no emit mode
 
-#### AngularJS
-
-- `test` - runs js tests in ServicePulse.Host.Test project
-- `setup` - this command runs few commands
-  - installs the npm packages
-  - runs `webpack`
-- `load` - this command does the same thing that `setup` but webpack is run only once to produce artifacts. This command is used by the builder
-- `serve` - serves js artifacts using `http-server`
-- `webpack` - runs webpack command, which does the following things for the given modules (monitoring, configuration)
-  - joins js together
-  - runs babel transpiler
-  - has a file watcher to run the above whenever file is saved
-- `lint` - checks with eslint all js files
-- `dev` - runs in parallel two scripts `webpack` and `serve`
-
-NOTE:
-Webpack observes files and updates the artifacts whenever they are changed, however at the moment not every bit of code is processed by webpack. Only monitoring and configuration-related files are.
+> [!NOTE]
+> Webpack observes files and updates the artifacts whenever they are changed; however, at the moment, not every bit of code is processed by Webpack. Only monitoring and configuration-related files are.
```

Comment:
I believe this section can also be removed, now that everything is in Vue, changes to any part of the code should be reloaded by the HMR module (hot module replacement)